README.md

```diff
@@ -120,6 +123,17 @@ In addition to the environment variable specified by `MaxMindDBEnv`, this
 module exports `MMDB_ADDR`, which contains the IP address used for lookups by
 the module. This is primarily intended for debugging purposes.
 
+## Client IP address lookup control ##
+
+In case you want supply your own value for the IP address to lookup, it may be
+done by setting the environment variable `MMDB_ADDR`.
+This can be done, for instance, with
+[ModSecurity](https://github.com/SpiderLabs/ModSecurity/) in (real) phase 1.
```

Comment:
Should we note that this doesn't actually work with ModSecurity out of the box (at least on most distros) and that you need to specially compile it? My primary objection to this change is that it is almost impossible to use this correctly.